roles/container-engine/containerd/tasks/main.yml

```diff
@@ -98,7 +98,9 @@
   meta: flush_handlers
 
 - name: containerd | Ensure containerd is started and enabled
-  service:
+  systemd:
     name: containerd
+    daemon_reload: yes
     enabled: yes
-    state: started
+    state: restarted
```

Comment:
Why `restarted`? This can cause an outage by restarting containerd instead of just ensuring it is properly started.

Reply:
In my tests it takes two runs through the role to get unmasked and started and enabled unless I changed state to restarted

I get that in general we don't want to restart already running daemons unless needed but in this case I don't see where it causes an outage. The outage has already been caused by this role at this point, since it stopped my running containerd and masked it (in the tasks above this one. https://github.com/kubernetes-sigs/kubespray/blob/master/roles/container-engine/containerd/tasks/main.yml#L8-L13)

Reply:
See the updated commit - moving the unmask allows reversion of the restart to start in my testing. So now this whole role does 1 restart instead of 2....  The second daemon_reload shouldn't logically be needed but experimentally is required and should be harmless